### DISCUSSION TITLE
Unable to follow steps in brew installer

### DISCUSSION THREAD

**Discussion by quackerex** [Homebrew/discussions#4696] [Getting started] [answered]
### Output of `brew config`

```shell
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
zsh: command not found: brew
```

### Description of issue

I can't run `brew shellenv` 

> To add Homebrew to your PATH run brew shellenv in your shell profile (e.g. ~/.bash_profile or ~/.zprofile)
> 	⁃	Run brew help to get started
> 	⁃	Further documentation:
> 	⁃	https://docs.brew.sh

**Comment by osalbahr** [comment, marked as answer]
This is a [known issue](https://github.com/orgs/Homebrew/discussions/4689) in the .pkg installer and will be fixed in the next release. For the meantime, you can run these commands and replace `homefolder` with your username (assuming you’re on an ARM-based macOS).

```
==> Next steps:
- Add Homebrew to your PATH in /Users/homefolder/.zprofile:
    echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
- Run `brew help` to get started
```

https://github.com/orgs/Homebrew/discussions/775#discussioncomment-359960
